src/addresstype.cpp

```diff
@@ -108,6 +109,11 @@ class CScriptVisitor
         return dest.GetScript();
     }
 
+    CScript operator()(const PubKeyDestination& dest) const
+    {
+        return CScript() << ToByteVector(dest.GetPubKey()) << OP_CHECKSIG;
```

Comment:
in https://github.com/bitcoin/bitcoin/pull/28246/commits/22d54175ab14e6f8f0426dc015b897a4c4a35284:

This line is causing the fuzzer to fail in `src/test/fuzz/script.cpp:155`, because it first calls `GetScriptForDestination`, (which now returns a script for PubKeDestination), and then calls `IsValidDestination` and checks if the `dest.empty()` and `IsValidDestination` match. Can probably just special case `PubKeyDestination`? Or maybe there's a more clever way to handle it

Reply:
This should do the trick (and seems reasonable): https://github.com/bitcoin/bitcoin/commit/7431452ab61a46715c31c8e0453234d328973088#diff-23790a106f16f165ad579a3966ae9faa871f148ca4640bc870f6071a121d5314L156-R161

Reply:
Fixed